x-pack/plugins/ingest_pipelines/public/application/components/pipeline_editor/components/processor_form/field_components/drag_and_drop_text_list.tsx

```diff
@@ -134,14 +136,16 @@ function DragAndDropTextListComponent({
                           justifyContent="center"
                           gutterSize="none"
                         >
-                          <EuiFlexItem grow={false}>
-                            <div
-                              {...provided.dragHandleProps}
-                              className="pipelineProcessorsEditor__form__dragAndDropList__grabIcon"
-                            >
-                              <EuiIcon type="grab" />
-                            </div>
-                          </EuiFlexItem>
+                          {!disableDragging && (
```

Comment:
Added a prop to make draggin optional, since for certain processors (like grok) the order matters but in others such as this one it doesnt.